# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/html/infrastructure/urls/resolving-urls/query-encoding/windows-1252.html.ini
# Commit: 85883a720dab
# Full Hash: 85883a720dab61e82bcdb7391236dc06427466ee
# Author: moz-wptsync-bot <wptsync@mozilla.com>
# Date: 2021-04-07 15:47:02
# Regressor Bug: 1702830
# File Overlap Count: 1
# Description:
#   Bug 1702830 - [wpt-sync] Update web-platform-tests to 336ec39f06d15863a4af6b2aa6592530b37a5aee, a=testonly
#   
#   MANUAL PUSH: wpt sync bot
#   
#   wpt-head: 336ec39f06d15863a4af6b2aa6592530b37a5aee
# ==============================================================================

diff -r acef12d74007 -r 85883a720dab testing/web-platform/meta/html/infrastructure/urls/resolving-urls/query-encoding/windows-1252.html.ini
--- a/testing/web-platform/meta/html/infrastructure/urls/resolving-urls/query-encoding/windows-1252.html.ini	Fri Apr 02 22:03:46 2021 +0000
+++ b/testing/web-platform/meta/html/infrastructure/urls/resolving-urls/query-encoding/windows-1252.html.ini	Fri Apr 02 22:06:52 2021 +0000
@@ -115,9 +115,11 @@
 
 [windows-1252.html?include=workers]
   expected:
-    if (os == "win") and debug and webrender and not swgl: ["TIMEOUT", "OK"]
-    if (os == "linux") and not webrender and debug: ["TIMEOUT", "OK"]
-    if (os == "mac") and debug: ["TIMEOUT", "OK"]
+    if (os == "linux") and (processor == "x86_64") and debug and webrender and not fission and not swgl: ["TIMEOUT", "OK"]
+    if (os == "linux") and (processor == "x86_64") and debug and not webrender: ["TIMEOUT", "OK"]
+    if (os == "linux") and (processor == "x86_64") and not debug and fission: ["TIMEOUT", "OK"]
+    if (os == "android") and debug and webrender: ["TIMEOUT", "OK"]
+    if (os == "linux") and (processor == "x86"): ["TIMEOUT", "OK"]
     [OK, TIMEOUT]
   [SharedWorker() in a shared worker]
     expected: FAIL
@@ -131,9 +133,11 @@
 
   [SharedWorker constructor]
     expected:
-      if (os == "win") and debug and webrender and not swgl: ["TIMEOUT", "PASS"]
-      if (os == "linux") and not webrender and debug: ["TIMEOUT", "PASS"]
-      if (os == "mac") and debug: ["TIMEOUT", "PASS"]
+      if (os == "linux") and (processor == "x86_64") and debug and webrender and not fission and not swgl: ["TIMEOUT", "PASS"]
+      if (os == "linux") and (processor == "x86_64") and not debug and fission: ["TIMEOUT", "PASS"]
+      if (os == "linux") and (processor == "x86_64") and debug and not webrender: ["TIMEOUT", "PASS"]
+      if (os == "android") and debug and webrender: ["TIMEOUT", "PASS"]
+      if (os == "linux") and (processor == "x86"): ["TIMEOUT", "PASS"]
       [PASS, TIMEOUT]
 
   [SharedWorker() in a dedicated worker]